src/edu/ucsb/nceas/metacat/doi/datacite/DefaultDataCiteFactory.java

```diff
@@ -0,0 +1,119 @@
+import java.text.SimpleDateFormat;
+
+import org.apache.log4j.Logger;
+import org.dataone.client.v2.itk.D1Client;
+import org.dataone.service.exceptions.InvalidRequest;
+import org.dataone.service.exceptions.ServiceFailure;
+import org.dataone.service.types.v1.Identifier;
+import org.dataone.service.types.v2.Node;
+import org.dataone.service.types.v2.ObjectFormat;
+import org.dataone.service.types.v2.SystemMetadata;
+import org.w3c.dom.Document;
+
+import edu.ucsb.nceas.ezid.profile.DataCiteProfile;
+import edu.ucsb.nceas.ezid.profile.DataCiteProfileResourceTypeValues;
+import edu.ucsb.nceas.ezid.profile.ErcMissingValueCode;
+import edu.ucsb.nceas.metacat.dataone.MNodeService;
+
+
+/**
+ * Default factory to generate a simple datacite metadata xml string
+ * @author tao
+ *
+ */
+public class DefaultDataCiteFactory extends DataCiteMetadataFactory {
+    private static Logger logMetacat = Logger.getLogger(DefaultDataCiteFactory.class);
+    
+    
+    /**
+     * Method to generate the data cite xml document
+     */
+    @Override
+    public String generateMetadata(Identifier identifier, SystemMetadata sysmeta) throws InvalidRequest, ServiceFailure {
+        if(identifier != null && sysmeta != null) {
+            try {
+                String language = "English";
+                Document doc = generateROOTDoc();
+                //identifier
+                String scheme = DOI;
+                String id = removeIdSchemePrefix(identifier.getValue(), scheme);
+                addIdentifier(doc, id, scheme);
+                
+                //creator
+                String affiliation = null;
+                String nameIdentifier = null;
+                String nameIdentifierSchemeURI = null;
+                String nameIdentifierScheme = null;
+                appendCreator(sysmeta.getRightsHolder().getValue(), doc, affiliation, nameIdentifier, nameIdentifierSchemeURI, nameIdentifierScheme);
+                
+                //title
+                appendTitle(ErcMissingValueCode.UNKNOWN.toString(), doc, language);
+                
+                //publish
+                Node node = MNodeService.getInstance(null).getCapabilities();
+                addPublisher(doc,node.getName());
+
+                //publication year
+                SimpleDateFormat sdf = new SimpleDateFormat("yyyy");
+                String year = sdf.format(sysmeta.getDateUploaded());
+                addPublicationYear(doc, year);
+                
+                // type
+                //String resourceType = lookupResourceType(sysmeta);
+                String resourceType = null; //only set the attribute to "dataset"
+                addResourceType(doc, DataCiteProfileResourceTypeValues.DATASET.toString(), resourceType);
+
+                // format
+                String format = lookupFormat(sysmeta);
+                if(format != null) {
+                   appendFormat(doc, format);
+                }
+                return serializeDoc(doc);
```

Comment:
I feel like returning an un-validated serialized document might cause us some headaches.  I'm wondering if this is a good place to load the DataCite schema and validate the document against it to make sure we are providing correct metadata, particularly for the DataCite version we are supporting.

Reply:
If the xml is invalid, the ezid server will reject it. So we don't need the duplicated validation.